Formula/clip.rb

```diff
@@ -0,0 +1,29 @@
+class Clip < Formula
+  desc "Create high-quality charts from the command-line"
+  homepage "https://clip-lang.org/"
+  url "https://github.com/asmuth/clip/archive/v0.5.tar.gz"
+  sha256 "39421ccb943ba8279de41e8dfbcf16f555aaa9b7a456054a8dcc65a995b68b5c"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "cairo"
+  depends_on "fmt"
+  depends_on "freetype"
+  depends_on "fribidi"
+  depends_on "harfbuzz"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+    pkgshare.install "examples", "tests"
+  end
+
+  test do
+    cp_r pkgshare/"examples", testpath
+    cp_r pkgshare/"tests", testpath
```

Comment:
You're not using these right?

Reply:
The examples reference data from `tests`.

https://github.com/asmuth/clip/blob/v0.5/examples/charts-basic/areachart.clp#L6-L7